index.html

```diff
@@ -5975,6 +5977,35 @@ <h2 id="ConvolverNode">
           </dd>
         </dl>
         <section>
+          <h2>
+            ConvolverOptions
+          </h2>
+          <p>
+            The specifies options for constructing a
+            <a><code>ConvolverNode</code></a>. All members are optional; if not
+            specified, the node is contructing using the normal defaults.
+          </p>
+          <dl title="dictionary ConvolverOptions : AudioNodeChannelOptions"
+          class="idl">
+            <dt>
+              AudioBuffer buffer
+            </dt>
+            <dd>
+              The desired buffer for the <a><code>ConvolverNode</code></a>.
+              This buffer will be normalized according to the value of
+              <code>disableNormalization</code>.
+            </dd>
+            <dt>
+              boolean disableNormalization = false
```

Comment:
Yes. I was following the suggestions and decisions made in https://github.com/WebAudio/web-audio-api/pull/545#issuecomment-112599527 wrt to the boolean trap and why we used `disableNormalization` for the `PeriodicWave` dictionary.